packages/cli/src/commands/generate.ts

```diff
@@ -0,0 +1,23 @@
+import { GluegunToolbox } from "gluegun";
+
+module.exports = {
+  name: "generate",
+  alias: ["g"],
+  run: async (toolbox: GluegunToolbox) => {
+    const {
+      parameters,
+      template: { generate },
+      print: { info }
+    } = toolbox;
+
+    const name = parameters.first;
```

Comment:
it generates undefined-model.ts if no filename provided

Reply:
yeah, it's just a showcase command, I will remove it later so I haven't mentioned it in documentation :)